src/api/utils/Trimmer.ts

```
@@ -0,0 +1,139 @@
+
+				console.log("cleared");
+
+				ctx.drawImage(img.image, 0, 0, img.width, img.height, 0, 0, img.width, img.height);
+
+				console.log("drawn");
+
+				const {data} = ctx.getImageData(0, 0, img.width, img.height);
+
+				console.log("got data");
+
+				spaces = this.getSpacing(data, img.width, img.height, threshold);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)